cancel_eur_subscription_auto.py

```
@@ -0,0 +1,137 @@
+
+        if not active_subs:
+            print(f"  ✅ No active subscriptions - ready for USD!")
+        else:
+            print(f"  ⚠️  Still has {len(active_subs)} active subscription(s)")
+            for sub in active_subs:
+                print(f"     - {sub.id}: {sub.currency.upper()}")
+    except Exception as e:
+        print(f"  ⚠️  Could not verify: {e}")
+
+    # Update database to reflect cancelled status
+    print(f"\n[4] Updating database...")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)